README.md

```diff
@@ -82,6 +82,8 @@ To contribute via pull request, follow these steps:
 2. Write your code, tests and documentation, and format them with ``black``
 3. Create a pull request describing your changes
 
+For more detailed instructions on how to contribute code, check out these code contributor guidelines. 
+
```

Comment:
@EmmaWightman this isn't actually linked -  I think you can add a [relative link](https://github.blog/2013-01-31-relative-links-in-markup-files/) to CONTRIBUTING.md

Reply:
thanks for spotting it @eliseboyd ! I updated the URL to one that should work correctly